packages/documentation-spa/sidebars.js

```diff
@@ -34,10 +35,11 @@ module.exports = {
       collapsed: false,
       items: [
         'microservices/api-gateway/api-gateway',
+        'microservices/apps-service/apps-service',
         'microservices/authorization/authorization-service',
```

Comment:
Let's put apps service after the authorization, since that is the prerequisite for all the microservices.
```suggestion
        'microservices/authorization/authorization-service',
        'microservices/apps-service/apps-service',
```

Reply:
Changed